src/components/accordion/_accordion.scss

```diff
@@ -70,15 +74,24 @@
     // This is styled to look like a link not a button
     .govuk-accordion__open-all {
       @include govuk-font($size: 16);
-      @include govuk-link-common;
-      display: inline;
+      position: relative;
```

Comment:
I am wondering whether the open all button meets colour contrast if the top-most section is already open when you go to focus it.

<img width="1009" alt="Screenshot 2019-05-10 at 09 45 05BST" src="https://user-images.githubusercontent.com/121939/57514426-6f8db000-7308-11e9-99aa-a59ebddbe6d1.png">

I think in this case the black underline 'blends in' with the blue top border, so I'm not sure it'd be perceivable, and the yellow alone does not meet contrast against the white page background.

@dashouse what do you think?

Reply:
@36degrees Blue to Black is 3.78:1. We're somewhat relying on this change being ok elsewhere. It also has the underline removed which is exclusive to the focus state. Go with it and see what comes up in the audit?

Reply:
Sounds good – I should've checked the contrast rather than just assuming, surprised it's that high! 👍